### DISCUSSION TITLE
Move a deleted alias to trash. Trash is emptied after 30 days.

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#387] [Feature Requests]
So user can restore a deleted alias. 

An alias in trash will have the same behavior as a disabled alias regarding receiving emails.

**Comment by nguyenkims** [comment]
Tracked in https://github.com/simple-login/app/issues/388

**Comment by CuerrytheMight** [comment]
Would also appreciate, if there was a trash and 30 days recovery time.

**Reply by hiccups1980** [reply]
This already exists.

**Reply by CuerrytheMight** [reply]
Ah yeah? That would be awesome, I wasn‘t able to find it though. Were is the trash (checked on iOS app and via browser on PC)..

**Reply by hiccups1980** [reply]
It works for subdomains and custom domains. 
For subdomains: Go to subdomains -> Details -> Deleted aliases.

**Reply by zachariah-at-msupply** [reply]
Does it work for standard aliases, or only aliases created with subdomains or domains?

**Comment by nguyenkims** [comment]
This feature is now available.
